src/EnergyPlus/SimAirServingZones.cc

```diff
@@ -1105,6 +1105,15 @@ namespace SimAirServingZones {
 								}
 							}
 						}
+						// added to fix bug issue #5695, if HW coil on outdoor air system, don't lock out during economizing
+						for (OASysNum = 1; OASysNum <= NumOASystems; ++OASysNum) {
+							for (OACompNum = 1; OACompNum <= OutsideAirSys( OASysNum ).NumComponents; ++OACompNum) {
+								CompType = OutsideAirSys( AirSysNum ).ComponentType( OACompNum );
```

Comment:
@Nigusse @EnergyArchmage Should this be `OutsideAirSys( OASysNum )` instead of `OutsideAirSys( AirSysNum )`?

I'm seeing some array extents assertion failures related to this line.

Reply:
@nealkruis I agree it should have been OASysNum.